docs/src/content/introduction/ssr.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 1
+title: Server Side Rendering
+category: introduction
+type: docs
+scope: null
+---
+
+#### Using Victory in Server Side Rendering Applications
```

Comment:
```suggestion
# Server-Side Rendering
```

As this is the main page heading, it should be a `h1` element rather than a `h4`- perhaps we title the page "Server Rendering"?